src/aliceVision/sfm/pipeline/global/GlobalSfMRotationAveragingSolver.hpp

```diff
@@ -22,6 +26,39 @@ enum ERelativeRotationInferenceMethod
   TRIPLET_ROTATION_INFERENCE_COMPOSITION_ERROR = 1
 };
 
+inline std::string ERotationAveragingMethod_enumToString(ERotationAveragingMethod eRotationAveragingMethod)
```

Comment:
please remove inline and split into hpp / cpp.
Also add doxygen doc for the functions